conf/ocsci/production-vsphere-upi-ocs4qe-lab.yaml

```diff
@@ -0,0 +1,8 @@
+---
+# This config will provide the 12 vCPU's to VM.
+# We are restricting to 12 due to hardware limitation in the lab
+DEPLOYMENT:
+  allow_lower_instance_requirements: true
+ENV_DATA:
+  worker_num_cpus: '12'
+  master_num_cpus: '4'
```

Comment:
Do we have enough memory to increase on our lab env or not really?

Reply:
We cannot accommodate 64GB